_specification-v1/components.md

```diff
@@ -92,11 +93,12 @@ Origami components **must** be installable through the <a href="https://npmjs.co
   - It **must** include a <a href="https://docs.npmjs.com/cli/v7/configuring-npm/package-json#dependencies">`dependencies`</a> property set to an object **_if_** the component has any production javascript dependencies which are not also Origami components.
   - It **must** include an <a href="https://docs.npmjs.com/cli/v7/configuring-npm/package-json#engines">`engines`</a> property set to an object which has an `npm` property set to a SemVer range which disallows anything below `v7.0.0`. E.G. `^7` would be valid as would `>= 7`.
   - It **may** contain a <a href="https://docs.npmjs.com/cli/v7/configuring-npm/package-json#description">`description`</a> property set to a short description of the component.
+  - It **may** contain a <a href="https://docs.npmjs.com/cli/v7/configuring-npm/package-json#keywords">`keywords`</a> property in order to help users discover the right component.
   - It **may** contain a <a href="https://docs.npmjs.com/cli/v7/configuring-npm/package-json#bugs">`bugs`</a> property set to the url of the issue-tracker for the component.
   - It **may** contain a <a href="https://docs.npmjs.com/cli/v7/configuring-npm/package-json#repository">`repository`</a> property set to the url of the git repository for the component.
   - It **may** contain a <a href="https://www.typescriptlang.org/docs/handbook/declaration-files/publishing.html#including-declarations-in-your-npm-package">`types`</a> property set to component's TypeScript declaration file.
   - It **may** contain a <a href="https://docs.npmjs.com/cli/v7/configuring-npm/package-json#homepage">`homepage`</a> property set to the url of the component on the <a href="https://registry.origami.ft.com/components">Origami Registry</a>, e.g. `"https://registry.origami.ft.com/components/o-table"`
-  - It **may** include an <a href="https://nodejs.org/api/packages.html#packages_conditional_exports">`exports`</a> property set to an object which a `browser` field which is set to the component's main JavaScript file (`main.js`) **_if_** it exists. e.g. `"exports":{"browser": "main.js"}`.
+  - It **may** include an <a href="https://nodejs.org/api/packages.html#packages_conditional_exports">`exports`</a> property set to an object which a `browser` field which is set to the component's main JavaScript file (`main.js`) **_if_** it exists. e.g. `"exports": {".": {"browser": "./main.js"}},`.
```

Comment:
I don't think we should add the exports object to the specification (If adding as a **may**) because it requires answering lots of questions around what paths a component should export and what paths to not export. I think it would be too much information for people to read for a part of the specification which is a **may**.

The exports object has a special feature, where it restricts the paths which can be imported from a project to only those defined in the exports object. 

If we only defined `.` in the exports object then the only import which is allowed would be for the component name. E.G. `import Table from 'o-table';` would work and `import FlatTable from 'o-table/src/js/Tables/FlatTable.js';` would not work.

Reply:
Perhaps we should move this to a tutorial. We assume that only main is imported currently, don't we? And should encourage a restrictive `exports`.

Reply:
Yes, a tutorial would be a better place for this 👍